pavement.py

```diff
@@ -549,8 +549,16 @@ def tarball_name(type='gztar'):
 
 @task
 def sdist(options):
+    # First clean the repo and update submodules (for up-to-date doc html theme
+    # and Sphinx extensions)
+    sh('git clean -xdf')
+    sh('git submodule init')
+    sh('git submodule update')
```

Comment:
I've certainly never needed that. Are you assuming that you may have local changes to the submodule code? That seems far less likely than changes to normal numpy code, which we also don't check for.